test/unit/unit/format.js

```diff
@@ -0,0 +1,152 @@
+define([
+	"cldr",
+	"src/core",
+	"src/unit/format",
+	"src/unit/get",
+	"json!cldr-data/main/en/units.json",
+	"json!cldr-data/supplemental/likelySubtags.json"
+], function( Cldr, Globalize, formatUnit, unitGet, enUnits, likelySubtags ) {
+
+var cldr, globalize;
+
+Cldr.load( enUnits );
+Cldr.load( likelySubtags );
+
+globalize = new Globalize( "en" );
+cldr = globalize.cldr;
+
+QUnit.module( "Unit Format" );
+
+function oneOrOtherPluralGenerator( plural ) {
+  if ( plural === 1 ) {
+    return "one";
+  } else {
+    return "other";
+  }
+}
+
+QUnit.assert.unitFormat = function ( value, unit, options, expected ) {
+	var compoundUnitPattern, form, unitProperties;
+
+	form = options.form || "long";
+	unitProperties = unitGet( unit, form, cldr );
+	compoundUnitPattern = unitGet( "per", form, cldr ).compoundUnitPattern;
+
+	this.equal(
+		formatUnit( value, unitProperties, oneOrOtherPluralGenerator, compoundUnitPattern ),
+		expected
+	);
+};
```

Comment:
Feel free to take or not this suggestion. But, I would remove the `"long"` default for `form` (in here, unit tests) and I would collapse both unit and compountUnitPattern in a single function `unit/properties.js` to avoid duplicating changes here and there.